script/openqa-bootstrap

```diff
@@ -14,6 +14,54 @@ if [ "$(ps --no-headers -o comm 1)" = 'systemd' ]; then
     running_systemd=1
 fi
 
+start-database() {
+    if [ -z "$running_systemd" ]; then
+        su postgres -c '/usr/share/postgresql/postgresql-script stop' || true
+        su postgres -c '/usr/share/postgresql/postgresql-script start'
+    else
+        systemctl enable --now postgresql
+    fi
+}
+
+start-worker() {
+    if [ -z "$running_systemd" ]; then
+        /usr/bin/install -d -m 0755 -o _openqa-worker /var/lib/openqa/pool/1
+        su _openqa-worker -c '/usr/share/openqa/script/worker --instance 1'
+    else
+        systemctl enable --now openqa-worker@1
+    fi
+}
+
+start-daemons() {
+    if [ -z "$running_systemd" ]; then
+        pgrep -f openqa-scheduler-daemon > /dev/null || su geekotest -c /usr/share/openqa/script/openqa-scheduler-daemon &
+        pgrep -f openqa-websockets-daemon > /dev/null || su geekotest -c /usr/share/openqa/script/openqa-websockets-daemon &
+        pgrep -f openqa-gru > /dev/null || su geekotest -c /usr/share/openqa/script/openqa-gru &
+        pgrep -f openqa-livehandler-daemon > /dev/null || su geekotest -c /usr/share/openqa/script/openqa-livehandler-daemon &
```

Comment:
These lines could actually be extracted into a function that only takes the e.g. `openqa-scheduler-daemon` as parameter.

Reply:
I left this for now